src/main/java/newbank/server/NewBankClientHandler.java

```diff
@@ -81,18 +53,8 @@ private boolean processRequest(final String request) {
     assert (tokens.length > 0);
 
     final String commandName = tokens[0].toUpperCase();
-    final Command command = getCommand(commandName, tokens);
-
-    if (tokens.length >= 2 && tokens[1].equalsIgnoreCase("HELP")) {
-      handleCommandHelp(command);
-      return true;
-    }
 
-    try {
-      out.println(command.execute());
-    } catch (CommandInvalidSyntaxException e) {
-      handleCommandInvalidSyntax(command);
-    }
+    out.println(getCommand(commandName, tokens).execute());
```

Comment:
Are you sure this doesn't break the help functionality? Shall we add a few assertions for it in the integration tests?

Reply:
Yep, kinda scary that there are not tests for this. I fiddled around with adding parameterized tests using reflection to get the syntax for each command, but I'm too tired to figure it out. :sweat_smile: 

But yes, I'm sure this doesn't break the help functionality! :D